release.sh

```diff
@@ -91,6 +84,10 @@ if [[ $devmode != 1 ]]; then
   # commit the postgres image
   container_id=$(docker ps  -f "name=postgres" -l -q)
   image_name=$package_name-postgres
+  # first we must stop postgres to avoid this error http://disq.us/p/230w4jl
+  echo "docker exec -u postgres $container_id pg_ctl stop"
+  docker exec -d -u postgres $container_id pg_ctl stop
+  sleep 15
```

Comment:
Sleep is required because we're running the command as detached `-d`. This is because `pg_ctl stop` does some CMD manipulation that messes with this `docker exec`, and causes the release.sh to exit without fully finishing. To get around this we exec, detach, and sleep.